lib/tasks/coverage_report.rake

```diff
@@ -0,0 +1,23 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+namespace :coverage do
+  desc "Collates all result sets generated by the different test runners"
+  task :report do
+    unless RUBY_VERSION >= '2.7.0'
+      puts 'This task requires Ruby 2.7.0+'
+      return
+    end
+    require 'simplecov'
+    require 'simplecov_json_formatter'
+    # FOR CI ONLY
```

Comment:
Instead of this comment, I think we can enforce the CI-only nature of this task by adding something like: 

```suggestion
unless CI
  puts 'This task is intended to be run only on the CI, exiting...'
  return
end
```

Reply:
Hi @kaylareopelle I like this edit, but couldn't figure out if we had a variable `CI` (the CI gave me: `NameError: uninitialized constant CI`). Could this be under something else? I tried a few potential name patterns.

Reply:
GHA (and others) will set a `CI` env var to signify that the code base is being evaluated in a CI context.

Where it says `if CI` now in pseudo code, you should be able to use `if ENV['CI']` with real Ruby code.